src/main/java/com/forgeessentials/economy/shop/ShopManager.java

```diff
@@ -305,16 +305,21 @@ public void playerInteractEvent(final PlayerInteractEvent event)
                 ChatOutputHandler.sendMessage(event.entityPlayer, msg);
                 return;
             }
+
             int removedAmount = 0;
+
             if (sameItem)
             {
                 removedAmount = Math.min(equippedStack.stackSize, transactionStack.stackSize);
                 equippedStack.stackSize -= removedAmount;
                 if (equippedStack.stackSize <= 0)
                     event.entityPlayer.inventory.mainInventory[event.entityPlayer.inventory.currentItem] = null;
             }
-            if (removedAmount < transactionStack.stackSize)
-                removedAmount += ModuleEconomy.tryRemoveItems(event.entityPlayer, transactionStack, transactionStack.stackSize - removedAmount);
+
+            if (removedAmount < transactionStack.stackSize) {
+                ModuleEconomy.tryRemoveItems(event.entityPlayer, transactionStack, transactionStack.stackSize - removedAmount);
```

Comment:
What is the reason for this change?

Reply:
removedAmount variable is not used in the neext code, so there is no point in the last change. Needless operation.

Reply:
This is an unnecessary change.  While it is true that removedAmount is not being used, there was no need to make this change as you did not change any fundamental logic and this just adds commit bloat.

Reply:
Ok, you're right. My bad.